scripts/create-sample-worksheet.py

```diff
@@ -0,0 +1,379 @@
+import argparse
+import os
+import random
+import string
+import sys
+import time
+
+sys.path.append('..')
```

Comment:
Why do you need this line?

Reply:
I originally had the util on the root level, but now it's not needed since I moved it to `scripts`. I updated.